docs/en/administrator-guide/operation/disk-capacity.md

```diff
@@ -131,12 +131,13 @@ When the disk capacity is higher than High Watermark or even Flood Stage, many o
 
     **This operation will affect [Restore data from BE Recycle Bin](./tablet-restore-tool.md).**
 
-    If the BE can still be started, you can use `CLEAN TRASH ON(BackendHost:BackendHeartBeatPort)` to actively clean up temporary files. There are two situations as follows: 
+    If the BE can still be started, you can use `CLEAN TRASH ON(BackendHost:BackendHeartBeatPort);` to actively clean up temporary files. **all trash files** and expired snapshot files will be cleaned up, **This will affect the operation of restoring data from the trash bin**.
```

Comment:
```suggestion
    If the BE can still be started, you can use `ADMIN CLEAN TRASH ON(BackendHost:BackendHeartBeatPort);` to actively clean up temporary files. **all trash files** and expired snapshot files will be cleaned up, **This will affect the operation of restoring data from the trash bin**.
```